Savala_Chris/Assignment/multiples#1.py

```diff
@@ -0,0 +1,8 @@
+for x in range(1,1001):
+	if(x % 2 ==1):
+		print x
+
+
+for v in range(5, 1000001):
```

Comment:
Good job. Keep the assignments (pull requests) coming. And feel free to group 2-3 assignments together if they're small enough. Otherwise 1 per branch / pull request as you've done.

Btw, there might be an easier to to solve the 2nd (% 5) problem here =]. Check through the docs for python's range, or alternatively, check out some of your classmates answers in their folders on my repo.